### PR TITLE
Fix speakers CSS

### PR DESCRIPTION
**Pull request by nhocki**
Before:

After:

### CODE REVIEW

File: docs/index.html [hunk 1]
```diff
@@ -195,7 +195,7 @@ <h4 class='speakers__title'>Speakers:</h4>
 </li>
 <li class='speakers__item'>
 <a class='speakers__link' href='https://twitter.com/jmcharnes' target='_blank'>
-<img alt='Jason Charnes' src='./assets/images/speakers/Jason-Charnes.jpg'>
+<img alt='Jason Charnes' src='./assets/images/speakers/Jason-charnes.jpg'>
```
Comment (sogamoso) [inline docs/index.html]: I think the change to image files shouln't be here, seems like the filenames are currently correct: https://github.com/RubyConfCo/rubyconfco-website-2017/tree/master/docs/assets/images/speakers

**Review by sogamoso — CHANGES_REQUESTED** [review]

File: docs/index.html (hunk 1 above)
Comment (nhocki) [reply docs/index.html]: Hmm idk, might as well just change them all to be lowercase?

2017-04-25 at 10 41 am

Comment (sogamoso) [reply docs/index.html]: That sounds like a good plan.

Comment (nhocki) [reply docs/index.html]: Done

**Review by sogamoso — APPROVED** [review]

**Pull request merged**